app/models/fault_report.rb

```diff
@@ -20,8 +20,25 @@ class FaultReport < ApplicationRecord
   belongs_to :reported_by,  class_name: 'User'
   belongs_to :fixed_by,     class_name: 'User', optional: true
 
-  enum severity: %I[annoying probably_worth_fixing show_impeding dangerous]
-  enum status: %I[reported in_progress cant_fix wont_fix on_hold completed]
+  enum severity: {
+    annoying: 0,
+    probably_worth_fixing: 1,
+    show_impeding: 2,
+    dangerous: 3
+  }
+
+  enum status: {
+    reported: 0,
+    in_progress: 1,
+    cant_fix: 2,
+    wont_fix: 3,
+    on_hold: 4,
+    completed: 5
+  }
```

Comment:
Is it maybe worth adding something for if there is actually no fault (e.g. user error?) or is it preferred to just delete the fault? Could argue it's worth tracking in case e.g. a false fault is reported more than once and it's indicative of something needing changing

Reply:
Fixed this, I think